CONTRIBUTING.md

```diff
@@ -96,15 +100,28 @@ You can go on Snowsight to perform manual inspection of the changes to your sche
 > * Your database will be named after your feature branch so it's easy to find on Snowsight. For example, if your feature branch is called
 > `snow-123-new-feature`, your database will be called `SYNAPSE_DATA_WAREHOUSE_DEV_snow_123_new_feature`.
 
+----
+
 ### 3. Manually Drop the Cloned Database (Optional)
 
-There is a second job in the repository (`drop_clone`) that will drop your branch's database clone once it has been merged into `dev`.
-In other words, once your cloned database is created for testing, it will remain open until your PR is closed (unless you manually drop it).
+It's important to note that once your cloned database is created for testing, it will remain open until your PR is closed (unless you manually drop it).
 
 An initial clone of the development database will not incur new resource costs, **HOWEVER**, when a clone deviates from the original
 (e.g. new changes to your schema are applied for testing), the cloned database will begin to incur costs the longer it exists in our warehouse.
 **Please be mindful of the amount of time your PR stays open**, as cloned databases do not get dropped until a PR is merged. For example, if your PR is open for >1 week, consider manually dropping your cloned database on Snowflake to avoid unnecessary cost.
 
+To manually drop your clone:
+
+1. Log onto Snowsight (the Snowflake web client)
+2. Navigate to `Databases` on the sidebar:
+
+   <img width="200" alt="image" src="https://github.com/user-attachments/assets/076f6a68-69e6-4072-8562-753769be1a93" />
+
+3. Navigate to your clone in the new panel, and hit `Drop` from the `...` dropdown:
+
+   <img width="939" alt="image" src="https://github.com/user-attachments/assets/3155b26b-ced0-44f6-8563-6da4daf9cf0e" />
+
+
```

Comment:
Does this work? The clone will be owned by `<DOMAIN>_ADMIN`, which I'm not sure at this point whether we will grant this role to developers.

Reply:
Yeah, this doesn't work unless you're <DOMAIN>_ADMIN. Unless you want to spend a little more time on the implementation design, I can rescope this PR to introduce a new PR label for manually triggering dropping the database (`drop_clone`). Otherwise I can make a ticket so we can think about this a little bit more, and remove this section entirely for now.

Reply:
`drop_clone` is an option. We could have a ruleset that gives priority to this label, so as long as its present the `test_with_clone` job doesn't run and the `drop_clone` job does run. Although I think it's a good rule of thumb that as long as a PR is open, the corresponding clone should be available for reviewers to poke around in. Closing a PR is a simple way to say "These changes are no longer open for review and the clone should be dropped".

Did you have a specific use-case in mind for dropping the clone while the PR is still open?

Reply:
> Did you have a specific use-case in mind for dropping the clone while the PR is still open?

No, this section was mainly to minimize cost, but we haven't had any cost issues from our current clone databases, so I'm leaning towards scrapping this section entirely for now.

> Although I think it's a good rule of thumb that as long as a PR is open, the corresponding clone should be available for reviewers to poke around in.

Agreed

Reply:
LMK what you think of [these edits](https://github.com/Sage-Bionetworks/snowflake/pull/193/commits/f2726602db81af2feca8ba863a700c7fbd93add1)